src/lib/rtc-to-net/rtc-to-net.ts

```diff
@@ -663,6 +670,19 @@ import ProxyConfig = require('./proxyconfig');
       return true;
     }
 
+    private calculateBandwidth_ = () : void => {
+      if (!this.stopBandwidthCalc_) {
```

Comment:
You're using `stopBandwidthCalc_` as a `boolean` so it should be one - however, take a look at how `initiateSnapshotting` terminates its loop. That will eliminate the need for this class-level variable altogether.